include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -150,7 +150,7 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
       }
       int64_t take = (std::min)(backup_count(), static_cast<int64_t>(count));
       set_backup_count(backup_count() - take);
-      count -= take;
+      count -= static_cast<int>(take);
```

Comment:
can you add a comment as to why this is safe?

Reply:
resolved.